.htaccess

```diff
@@ -119,6 +122,7 @@
 	RewriteRule ^(.*)/topics/authentication$ /$1/developer_guides/security/authentication [R=301,L]
 	RewriteRule ^(.*)/topics/caching$ /$1/developer_guides/performance/caching [R=301,L]
 	RewriteRule ^(.*)/topics/commandline$ /$1/developer_guides/cli [R=301,L]
+	RewriteRule ^/topics/datamodel/?$ /en/developer_guides/model/data_model_and_orm  [R=301,L]
```

Comment:
Any 404s that include the /en/ part will need to follow the regex format with ^(.*) etc and $1 in the destination.